sdks/typescript/src/apache_beam/transforms/pardo.ts

```diff
@@ -85,11 +85,7 @@ export interface DoFn<InputT, OutputT, ContextT = undefined> {
 /**
  * Creates a PTransform that applies a `DoFn` to a PCollection.
  */
-export function parDo<
-  InputT,
-  OutputT,
-  ContextT extends Object | undefined = undefined,
->(
+export function parDo<InputT, OutputT, ContextT = undefined>(
```

Comment:
Is this a prettier change too?

Reply:
no, this is a TS change to make it compile

Reply:
Oh, I'm seeing the change now.